src/commands/script.rs

```diff
@@ -49,25 +49,23 @@ impl Script {
             if consuming_help {
                 if line.starts_with("# END HELP") {
                     consuming_help = false;
-                } else if line.starts_with("# ") {
+                } else if let Some(rest) = line.strip_prefix("# ") {
                     // omit first two characters since they are
                     // signifying continued help.
-                    help_string.push_str(&line[2..]);
-                }
-            } else {
-                if line.starts_with("# SOURCE") {
-                    should_source = true;
-                } else if line.starts_with("# START HELP") {
-                    consuming_help = true;
-                } else if line.starts_with("# SUMMARY: ") {
-                    // 9 = prefix, -1 strips newline
-                    summary_string.push_str(&line[11..(line.len() - 1)]);
-                } else if !line.starts_with("#!") {
-                    // if a shebang is encountered, we skip.
-                    // as it can indicate the command to run the script with.
-                    // metadata lines must be consecutive.
-                    break;
+                    help_string.push_str(rest);
                 }
+            } else if line.starts_with("# SOURCE") {
+                should_source = true;
+            } else if line.starts_with("# START HELP") {
+                consuming_help = true;
+            } else if line.starts_with("# SUMMARY: ") {
```

Comment:
nitpick: should this also use strip_prefix? IIUC the above strip_prefix was introduced to eliminate the hand-counting of a starting index for push_str.

Reply:
Good nit :). But if you're ok w/ it I think I already made changes to this in Batch #2 and prefer that we nit it over there 🗡️